core/serv/src/main.rs

```diff
@@ -410,25 +414,82 @@ async fn me(id: Identity) -> impl Responder {
     web::Json(id)
 }
 
+fn set_logging(log_level: &str, log_dir: &Path) -> Result<()> {
+    use flexi_logger::{
+        style, AdaptiveFormat, Age, Cleanup, Criterion, DeferredNow, Duplicate, LogSpecBuilder,
+        LogSpecification, Logger, Naming, Record,
+    };
+
+    fn log_format(
+        w: &mut dyn std::io::Write,
+        now: &mut DeferredNow,
+        record: &Record,
+    ) -> Result<(), std::io::Error> {
+        write!(
+            w,
+            "[{} {:5} {}] {}",
+            DateTime::<Utc>::from(*now.now()).to_rfc3339_opts(SecondsFormat::Secs, true),
+            record.level(),
+            record.module_path().unwrap_or("<unnamed>"),
+            record.args()
+        )
+    }
+
+    fn log_format_color(
+        w: &mut dyn std::io::Write,
+        now: &mut DeferredNow,
+        record: &Record,
+    ) -> Result<(), std::io::Error> {
+        let level = record.level();
+        write!(
+            w,
+            "[{} {:5} {}] {}",
+            DateTime::<Utc>::from(*now.now()).to_rfc3339_opts(SecondsFormat::Secs, true),
+            style(level, level),
+            record.module_path().unwrap_or("<unnamed>"),
+            record.args()
+        )
+    }
+
+    let log_spec = LogSpecification::env_or_parse(log_level)?;
+    let log_spec = LogSpecBuilder::from_module_filters(log_spec.module_filters())
+        .module("actix_web::middleware::logger", log::LevelFilter::Warn)
```

Comment:
WMiddleware should be @ `warn` level only when the default setting is used.
Otherwise it should be taken from RUST_LOG.
Pls remove this line, and revert the original workaround below.